docs/toolchain/freebsd/index.rst

```diff
@@ -34,3 +34,34 @@ Then go ahead and configure, build and install using ``meson``:
    Interfaces; libaio, liburing, and libvfn are not supported on FreeBSD.
 
 
+FreeBSD (14)
+~~~~~~~~~~~~
+
+Install the required toolchain and libraries, with sufficient system privileges
+(e.g. as ``root`` or with ``sudo``), by executing the commands below. You can
+run this from the root of the **xNVMe** by invoking::
+
+  sudo ./xnvme/toolbox/pkgs/freebsd-14.sh
```

Comment:
If it's from "the root of xNVMe", there shouldn't be a `./xnvme` in path, right?

Reply:
Nice catch!

Reply:
You still have 
```
You can run this from the **xNVMe** repository by invoking::

  sudo ./xnvme/toolbox/pkgs/freebsd-13.sh
```

If it's "from" the xNVMe repo, shouldn't the command just be `sudo ./toolbox (...)` or is this some kind of convention I don't know of? :)

Reply:
yeah. Do you have suggestion on how to fix the phrasing without changing the command?

Reply:
How about this...